src/components/sequencing/form/EnumEditor.svelte

```diff
@@ -10,19 +10,20 @@
   const MAX_SEARCH_ITEMS = 1_000;
 
   export let argDef: FswCommandArgumentEnum;
-  export let commandDictionary: CommandDictionary;
+  export let commandDictionary: CommandDictionary | null = null;
   export let initVal: string;
   export let setInEditor: (val: string) => void;
+  export let useQuotes: boolean = true;
 
   let enumValues: string[];
   let isValueInEnum: boolean = false;
   let value: string;
 
-  $: value = unquoteUnescape(initVal);
-  $: enumValues = commandDictionary.enumMap[argDef.enum_name]?.values?.map(v => v.symbol) ?? argDef.range ?? [];
+  $: value = useQuotes ? unquoteUnescape(initVal) : initVal;
```

Comment:
We haven't been showing quotes in the drop down since they weren't strictly needed. Now that we support symbolic values showing quotes could distinguish between symbols and literals.

@lklyne what is your take on quoting values in drop down boxes to mirror the required syntax in the editor?

+ Editor and Form more similar, clarity on symbol vs literal values
- Reduced readability for most common use case

Reply:
My thought is that if we display the `"` in the dropdown, then it might inform users that required quotes must be explicitly entered in regular text inputs. I think that the user shouldn't necessarily have to worry too much about correct syntax when using the form.

Reply:
I think I might even go as far as to say that `EnumEditor` shouldn't even know about `useQuotes`. I think the `ArgEditor` component should be in charge of formatting the output in the `setInEditor` callback.

Reply:
I agree that the EnumEditor code is simplified if we use quoted values within. Subjectively, I prefer no quotes from an aesthetic point of view.

I think the a more compelling reason for no quotes is to better align with the command dictionary. 

The need for quotes in the editor is due to the SeqN and VML language specs. In the form view it's unambiguous unless we decide to include both symbols and literals in the selectable set without other context clues
e.g.

- OPTION_A (dictionary literal)
- OPTION_B (dictionary literal)
- FOOBAR (variable)